src/Common/test-runtime/LatestDependencies/project.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
@roncain Basically we will need to change the validation rule to allow different versions and * in corefx build script dirs.prop.